spec/vcr_configurer.rb

```diff
@@ -10,6 +10,56 @@ class VcrConfigurer
       VCR_PROCESS_ID = 'VCRFakeProcessId'
       @ignore_vcr_requests = false
 
+      def self.setup
+        gdc_path_matcher = lambda do |client_request, recorded_request|
+          client_path = client_request.parsed_uri.path
+          recorded_path = recorded_request.parsed_uri.path
+          return true if matches_project_cache_fake_id?(client_path, recorded_path)
+
+          uploads_regex = %r{(\/gdc\/uploads\/)[^\/]+(.*)}
+          if client_path.match(uploads_regex) && recorded_path.match(uploads_regex)
+            # custom path matcher, sanitizing the randomness of uploads dirs
+            client_path.gsub(uploads_regex, '\1UPLOADS_TMP\2') == recorded_path.gsub(uploads_regex, '\1UPLOADS_TMP\2')
+          else
+            client_path == recorded_path
+          end
+        end
+
+        VCR.configure do |vcr_config|
+          vcr_config.cassette_library_dir = 'spec/integration/vcr_cassettes'
+          vcr_config.hook_into :webmock
+          vcr_config.allow_http_connections_when_no_cassette = true
+          vcr_config.configure_rspec_metadata!
+
+          vcr_config.ignore_request do
+            @ignore_vcr_requests
+          end
+
+          vcr_config.default_cassette_options = {
+            :decode_compressed_response => true,
+            :match_requests_on => [gdc_path_matcher, :method, :query],
+            # allow to set record mode from environment, see https://relishapp.com/vcr/vcr/v/3-0-3/docs/record-modes
+            :record => vcr_record_mode
+          }
+
+          %w(request response).each do |part|
```

Comment:
warning: Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`. (rubocop)